modules/tensor_mechanics/src/materials/ComputeIncrementalSmallStrain.C

```diff
@@ -55,7 +57,9 @@ ComputeIncrementalSmallStrain::computeProperties()
 
     A -= Fbar; // A = grad_disp - grad_disp_old
 
-    _strain_increment[_qp] = 0.5*(A + A.transpose());
+    RankTwoTensor total_strain_increment = 0.5*(A + A.transpose());
```

Comment:
Here and in one other place you've replaced 1 line with 2 lines that appears to me to do the same thing?  Was that intended?